src/create.cpp

```diff
@@ -40,27 +42,61 @@ unsigned Create::run()
         throw rest::Error(ret.error());
     }
 
-    std::string assetJson = m_request.body();
-
-    auto ret = AssetManager::createAsset(assetJson, user.login());
-    if (!ret) {
-        auditError(ret.error());
-        throw rest::errors::Internal(ret.error());
+    cxxtools::SerializationInfo si;
+    std::stringstream           jsonIn;
+    try {
+        jsonIn << m_request.body();
+        cxxtools::JsonDeserializer deserializer(jsonIn);
+        deserializer.deserialize(si);
+    } catch (const std::exception& e) {
+        logError(e.what());
+        return HTTP_CONFLICT;
```

Comment:
Why conflict? It should be internal error? No?

Reply:
Clement told me to do similar to delete.cpp. There you can find 'return HTTP_CONFLICT;'

Reply:
But this is a strange...
@perrettecl please, resolve this :)